### PR TITLE
Tests in Typescript

### PR DESCRIPTION
**Pull request by Alan-Cha**
Signed-off-by: Alan Cha <Alan.cha1@ibm.com>

### CODE REVIEW

**Comment by Alan-Cha** [comment]
The servers (and the evaluation) are still in Javascript though. We can iterate on it.

File: packages/openapi-to-graphql/test/authentication.test.ts [hunk 1]
```diff
@@ -147,12 +147,13 @@ test('Get project using API key passed as option - viewer is disabled', async ()
...
 test('Get project using API key passed in the requestOptions - viewer is disabled', async () => {
-  let { schema } = await openapiToGraphql.createGraphQlSchema(oas, {
+  const { schema } = await openapiToGraphql.createGraphQlSchema(oas, {
     viewer: false,
     requestOptions: {
       headers: {
         access_token: 'abcdef'
-      }
+      },
+      url: undefined
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/test/authentication.test.ts]: This is kind of ugly... not sure if there's much we can do about it. We rely on the request module for this object type definition.

File: packages/openapi-to-graphql/test/example_api.test.ts [hunk 2]
```diff
@@ -841,12 +842,12 @@ test('Capitalized enum values can be returned', () => {
...
 
 test('Define header and query options', () => {
-  let options = {
+  const options = {
     headers: {
       exampleHeader: 'some-value'
     },
     qs: {
-      limit: 30
+      limit: '30'
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/test/example_api.test.ts]: Typo caught because of Typescript

Comment (ErikWittern) [reply packages/openapi-to-graphql/test/example_api.test.ts]: Nice!

File: packages/openapi-to-graphql/test/authentication.test.ts (hunk 1 above)
Comment (ErikWittern) [reply packages/openapi-to-graphql/test/authentication.test.ts]: That's an ok price to pay, IMO.

**Comment by ErikWittern** [comment]
@Alan-Cha Nice, I am super happy we are using TypeScript in tests now, because it (likely) more closely emulates how OtG is used.

**Pull request merged**